app/src/main/res/drawable/ic_baseline_lock_24.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+  android:width="80dp"
+  android:height="80dp"
+  android:autoMirrored="true"
+  android:tint="#FFFFFFFF"
```

Comment:
Is this a core android icon?
We can use dimens.xml , color.xml res file for values.

Reply:
I got it in vector assets. Other `ic_` vector images don't use dimens.xml , color.xml. Is that a convention?